_data/banking.yml

```diff
@@ -260,6 +260,14 @@ websites:
       software: Yes
       doc: https://www.dnb.no/privat/kundeservice/innlogging-nettbank.html
 
+    - name: Dutch Bangla Bank Limited (DBBL)
```

Comment:
Looking at their website they appear to hyphenate Dutch-Bangla. So in full it is `Dutch-Bangla Bank Limited`.